### PR TITLE
add and update readmes

### LINKED ISSUE #1: fix readme file descriptions of data

### PR DESCRIPTION
**Pull request by elray1**
This would resolve #1 and #18.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+
 Influenza forecasting using data fusion
 
+## Repository organization
+
+This repository has the following directories:
+
+- `code/`: all code for flusion and its component models, exploratory data analyses, and so on. See the readme in that folder for further information.
+- `data-raw/`: raw data measuring influenza activity, pulled from various sources. See the readme in that folder for further information.
+- `retrospective-hub/`: a folder with the same structure as the FluSight forecast hub, used for storing model outputs generated in retrospective analyses.
```
Comment (elray1) [inline README.md:11]: This isn't committed yet, but will be there in a near-future PR.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
 # flusion
+
 Influenza forecasting using data fusion
 
+## Repository organization
+
+This repository has the following directories:
+
+- `code/`: all code for flusion and its component models, exploratory data analyses, and so on. See the readme in that folder for further information.
+- `data-raw/`: raw data measuring influenza activity, pulled from various sources. See the readme in that folder for further information.
```
Comment (lshandross) [inline README.md:10]: noticed that the `data-raw/` directory doesn't have a readme itself, though its sub-folders of each source do. Is this intentional?

File: code/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+This directory contains all code for the flusion model, organized into the following subdirectories:
+
+- `data-pipeline`: Code used for loading and normalizing the data contained in `flusion/data-raw`. This is installed as a python module in the project's Conda environment.
+- `eda`: A python notebook with exploratory data analyses exploring relationships between the input data signals.
+- `eval`: R scripts with informal evaluations that I ran each week throughout the season.  These are likely not of interest to others.
+- `flusion`: R scripts used to compute the flusion submission as an ensemble of the `gbq_qr`, `gbq_qr_no_level`, and `sarix` models, and to visualize the predictions.
+- `gbq`: Python code for running the GBQ models.
+- `sarix_model`: A notebook used to generate sarix model predictions.
+
```
Comment (lshandross) [inline code/README.md:11]: Why isn't there a bullet point describing the `glg` directory?

**Comment by elray1** [comment]
Thanks for the comments, @lshandross! I added some content to the code readme and added a new data-raw readme.

**Pull request merged**